### ISSUE TITLE
How to change FeaturePlot expression scale to z-score transformed scale

### ISSUE THREAD

**Issue by ksaunders73** [satijalab/seurat#4543]
Hello! 

This may be because I do not fully understand the source code for FeaturePlot() (https://github.com/satijalab/seurat/blob/master/man/FeaturePlot.Rd), but by what I think I understand, the scale on the righthand side is in terms of relative expression of a feature to the maximum expression of that feature (to the whole dataset or to the clusters or to specified conditions if they are specified?). 

**Is there a way to transform this relative expression into a Z score (or some other statistic) and make the scale show these Z scores?**

I am also running some code from https://github.com/satijalab/seurat/issues/3521 which allows me to plot a list of genes onto a single Feature Plot:

```
nkfocuslist <- list(c("FOS", "TNFAIP3", "NR4A1", "CCL4", "RASGRP2",
"PARVB", "NR4A2"))
sobj <- AddModuleScore(object = sobj, features = nkfocuslist, name = "NK_Focus_List")
FeaturePlot(object = sobj, features = "NK_Focus_List1")
```
![image](https://user-images.githubusercontent.com/62159599/119862178-da08ed00-bedd-11eb-96eb-8678541d6917.png)

**Is there a way to make this scale a Z-score scale as well?**

Hopefully this makes sense, and thank you for reading!

**Comment by yuhanH** [comment]
Hi, 
This ModuleScore looks a little wired in your plot. Have you run `NormalizeData` before `AddModuleScore`?
Since module score stores in the meta.data,  you can convert it to the Z-score directly. 
```
sobj$NK_Focus_List1 <- (sobj$NK_Focus_List1  - mean(sobj$NK_Focus_List1 )) / sd(sobj$NK_Focus_List1 )
```

**Comment by ksaunders73** [comment]
Hello @yuhanH ! This worked! Thanks so much for your help!! As for my data, it is a TPM matrix so I skipped the normalization step as I've seen recommended on this forum (https://github.com/satijalab/seurat/issues/668) and the bioinformatics stack exchange (https://bioinformatics.stackexchange.com/questions/5115/seurat-with-normalized-count-matrix). I only ran FindVariableFeatures() (ExpMean), ScaleData() (default linear regression), and then RunPCA(), FindNeighbors(), FindClusters(), and then the RunUMAP().
